pkg/blockstorage/awsefs/awsefs.go

```diff
@@ -112,7 +112,8 @@ func (e *Efs) Type() blockstorage.Type {
 // volume info that is sent back from the AWS EFS.
 func (e *Efs) VolumeCreate(ctx context.Context, volume blockstorage.Volume) (*blockstorage.Volume, error) {
 	req := &awsefs.CreateFileSystemInput{}
-	req.SetCreationToken(uuid.NewV4().String())
+	reqId, _ := uuid.NewV4()
```

Comment:
We should not ignore the error here and at other places right?

Reply:
Correct. It should be checked and when not nil, wrap and return.
Same below.

Reply:
I agree. This is production code, we should not ignore errors.